src/app/_components/BlockList/LayoutA/BlockCount.tsx

```diff
@@ -0,0 +1,46 @@
+import { useColorModeValue } from '@chakra-ui/react';
+import pluralize from 'pluralize';
+import { memo } from 'react';
+import { PiArrowUpRight } from 'react-icons/pi';
+
+import { Circle } from '../../../../common/components/Circle';
+import { ExplorerLink } from '../../../../common/components/ExplorerLinks';
+import { Flex } from '../../../../ui/Flex';
+import { Icon } from '../../../../ui/Icon';
+import { Text } from '../../../../ui/Text';
+
+export const BlockCount = memo(function ({ count }: { count: number }) {
+  const bgColor = useColorModeValue('purple.100', 'slate.900');
```

Comment:
I'm seeing a lot of this: defining the color with hooks. Not a huge fan of this as it creates a lot of noise that gets mixed in with the functional logic. Not suggesting we do things differently rn but I think it would be worth it at some point to find a different solution that seperates this logic out of the component would be more manageable.

Reply:
I agree, for the default color configuration, we add them to the theme. However, these are specific to the component. LMK if you have any suggestions. Maybe we could move them to a hook that is collocated with each component?